package.json

```diff
@@ -91,14 +91,14 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
+    "jest": "jest",
     "lint:js": "eslint .",
     "lint:md": "remark . --quiet --frail",
     "lint": "npm-run-all --parallel lint:*",
     "pretest": "npm-run-all --serial lint flow",
     "release": "npmpub",
-    "test:jest": "jest",
-    "test": "npm-run-all --serial test:*",
-    "watch": "npm run test:jest -- --watch"
+    "test": "jest",
+    "watch": "npm run jest -- --watch"
```

Comment:
Done in https://github.com/stylelint/stylelint/pull/2172/commits/ff6acfed29dc2c5cb3b6f4bfd5be756ad2588207 :)